dogstatsd.py

```diff
@@ -474,10 +479,47 @@ def run(self):
                 sys.exit(AgentSupervisor.RESTART_EXIT_STATUS)
 
     @classmethod
-    def info(self):
+    def info(self, cfg=None):
         logging.getLogger().setLevel(logging.ERROR)
+        if cfg and cfg.get('dogstatsd6_enable', False):
+            dsd6_status = Dogstatsd._get_dsd6_stats(cfg)
+            if dsd6_status:
+                dsd6_status.render()
+                return 0
+            else:
+                return -1
```

Comment:
can we also print something here when dogstatsd6 is enabled but can't be reached/its response is invalid? No need to format it very nicely

Same thing for the flare